bin/bin.js

```diff
@@ -150,7 +150,7 @@ const main = async () => {
     getFolderName({ feed, template: outDir })
   );
 
-  if (!fs.existsSync(basePath)) {
+  if (!fs.existsSync(basePath) && !info && !list) {
```

Comment:
Instead of adding these booleans here, what do you think about moving the existing code down below following conditional:

```js
if (info || list) {
  process.exit(0);
}

// move here!
```